applications/StructuralMechanicsApplication/python_scripts/check_postprocess_eigenvalues_file_process.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+# Import KratosUnittest
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+# Other imports
+import os
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
+    return CheckPostprocessEigenvaluesFileProcess(Model, settings["Parameters"])
+
+class CheckPostprocessEigenvaluesFileProcess(KratosMultiphysics.Process, KratosUnittest.TestCase):
+    """
+    This process compares the *.post.res file written by the PostprocessEigenvaluesProcess
```

Comment:
This class assumes a certain results format. Can you give a ref to the format or to the class that creates the results file expected by this class?